app/javascript/mastodon/components/status_content.jsx

```diff
@@ -220,7 +220,7 @@ class StatusContent extends React.PureComponent {
 
     const hidden = this.props.onExpandedToggle ? !this.props.expanded : this.state.hidden;
     const renderReadMore = this.props.onClick && status.get('collapsed');
-    const renderTranslate = this.props.onTranslate && status.get('translatable');
+    const renderTranslate = this.props.onTranslate && status.get('translation_languages').includes(intl.locale);
```

Comment:
I think a case-insensitive comparison is needed, e.g. DeepL returns `en-gb` but `intl.locale` is `en-GB`

Reply:
True. I have fixed the DeepL service to return normalized locale strings, i.e. `en-GB`.

The translation controller currently uses `content_language` i.e. only the first part of the locale string, so I changed the JavaScript to use the same approach. We might want to do something more clever at some point.